### DISCUSSION TITLE
An error occurred when I use the “GET /api/artifacts/{id}/data/**”

### DISCUSSION THREAD

**Discussion by fcm2000105** [International-Data-Spaces-Association/DataspaceConnector#718] [Q&A]
I have ConnectorA (https://localhost:8089/api/ids/data)  acting as provider and connectorB (https://localhost:8080/api/ids/data)  acting as consumer.
I got the right response body when negotiating a Contract Agreement with “Post /api/ids/contract”.
![image](https://github.com/International-Data-Spaces-Association/DataspaceConnector/assets/124911140/5d4e415f-dcf4-4a19-862b-2fb52cf3d5a5)
![image](https://github.com/International-Data-Spaces-Association/DataspaceConnector/assets/124911140/91fc01f9-7161-4d3f-ba3f-e0304d799ad1)
But an error occurred when I use “GET /api/artifacts/{id}/data/**” to access the data.
![image](https://github.com/International-Data-Spaces-Association/DataspaceConnector/assets/124911140/63ef5f14-4cc3-4ef8-994a-5da02e4f8ac4)
![image](https://github.com/International-Data-Spaces-Association/DataspaceConnector/assets/124911140/b33032d5-995a-4e14-8afd-692a7f3c9be0)
The error code is as follows:
[issues.txt](https://github.com/International-Data-Spaces-Association/DataspaceConnector/files/11756873/issues.txt)

**Comment by Halfold** [comment]
Yes, i got the same problem.
org.postgresql.util.PSQLException: Large Objects may not be used in auto-commit mode.
By the way, if I use H2 Database in my Connector, everything work smoothly and no mistakes. But PostgreSQL databases always get strange errors.
